dss-server/src/test/java/io/github/ztkmkoo/dss/server/rest/DssRestExceptionHandlerTest.java

```diff
@@ -0,0 +1,128 @@
+package io.github.ztkmkoo.dss.server.rest;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceRequest;
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceResponse;
+import io.github.ztkmkoo.dss.core.actor.rest.service.DssRestActorJsonService;
+import io.github.ztkmkoo.dss.core.exception.annotation.ExceptionHandler;
+import io.github.ztkmkoo.dss.core.exception.annotation.ServiceExceptionHandler;
+import io.github.ztkmkoo.dss.core.exception.handler.DssExceptionHandler;
+import io.github.ztkmkoo.dss.core.message.rest.DssRestServiceActorCommandRequest;
+import io.github.ztkmkoo.dss.core.network.rest.enumeration.DssRestMethodType;
+import lombok.Getter;
+import lombok.Setter;
+import org.junit.jupiter.api.Test;
+
+import java.io.Serializable;
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DssRestExceptionHandlerTest {
+
+    @Test
+    void test() throws InterruptedException {
+        final DssRestServer dssRestServer = new DssRestServer("127.0.0.1", 8181);
+        dssRestServer
+                .addDssService(new MyService())
+                .addDssService(new YourService())
+                .addExceptionHandler(new GlobalExceptionHandler());
+
+        stopDssRestServerAfterActivated(dssRestServer, 10 ,15);
+
+        dssRestServer.start();
+
+        assertTrue(dssRestServer.isShutdown());
+    }
```

Comment:
@Alencion 
This test looks a little bit strange. Could you tell me the detail?

Reply:
I want to test that the server operates after addExceptionHandler is added.

Reply:
Yap, i see.

But the Test class name is `DssRestExceptionHandlerTest.java`. 
I was mistaken that you test the `ExceptionHandler.java` interface.
You already test it in `DssRestServerTest.java` why test here?